### PR TITLE
Update - Add missing text to borgmatic backup tutorial

### PR DESCRIPTION
**Pull request by jasonnab**
Two simple small changes, to ensure proper readability and understanding of the [borgmatic article](https://community.hetzner.com/tutorials/install-and-configure-borgmatic/).

I have read and understood the Contributor's Certificate of Origin available at the end of 
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Jason N. <githubjn@nabein.me>

### CODE REVIEW

File: tutorials/install-and-configure-borgmatic/01.en.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ After the successful initialization we can create our first backup.
 borgmatic --verbosity 1
 ```
 
-The first backup may take a while, depending on the of your Server.
+The first backup may take a while, depending on the size of your Server.
```
Comment (fauust) [inline tutorials/install-and-configure-borgmatic/01.en.md:110]: I am not sure about this.
"Server size" is confusing. Maybe "depending on the size of the data to backup".

Comment (jasonnab) [reply tutorials/install-and-configure-borgmatic/01.en.md:110]: I agree with this upon reviewing the text overall. My original intention was to minimize the changes done, but did feel it sounded weird to refer to the size of the server, versus an alternative wording.

Comment (rook59) [reply tutorials/install-and-configure-borgmatic/01.en.md:110]: > My original intention was to minimize the changes done

Change as much as you like!

**Review by rook59 — APPROVED** [review]

**Comment by leojakeson** [comment]
Thank you very much!

**Pull request merged**